app/views/applications/show.html.erb

```diff
@@ -0,0 +1,48 @@
+<h1>Application</h1>
+<ul>
+    <li>Applicant's Name: <%= @application.name %></li>
+    <li>Street Address: <%= @application.street_address %></li>
+    <li>City: <%= @application.city %></li>
+    <li>State: <%= @application.state %></li>
+    <li>Zipcode: <%= @application.zip_code %></li>
+    <li >Name of pet(s): 
+        <% @application.pets.each do |pet| %>
+            <ul>
+            <li id="pet_name_<%=pet.id%>"><%= link_to "#{pet.name}", "/pets/#{pet.id}" %></li>
+            </ul>
+        <%end%>
+    </li>
+    <li>Reason why you want to adopt: <%= @application.description %></li>
+
+    <li>Status of Application: <%= @application.status %></li><br>
+    <div> 
+        <%if @application.in_progress? %>
+            <%= form_with url: "/applications/#{@application.id}", method: :get do |form|%>
+                <%= form.label :pet_name, "Add a Pet to this Application:" %>
+                <%= form.text_field :pet_name %>
+                <%= form.submit "Search" %>
+            <% end %><br>
+
+            <% if @pets %>
+                Search Result:
+                    <% @pets.each do |pet| %>
+                    <div id="pet_<%=pet.id%>">
+                      <li><%= link_to "#{pet.name}", "/pets/#{pet.id}"%> 
+                      <%= button_to "Adopt this Pet", "/applications/#{@application.id}/pets",  method: :post, params:{pet_key: pet.id}%>
+                      </li>
+                      <br>
+                     </div>
+                <%end%>
+            <%end%>
+        
+
+            <% if @application.pets.size > 0 %>
```

Comment:
This is very minor and is the only MVC infraction I've seen (and the rubric allows for 1-2), but I think it would be best to encapsulate this into a model method, and you could name it something like `submittable?` or `ready_to_submit?` to increase readability of this line. This is technically a data calculation (could use .count to make it just one SQL query) so is best implemented in a model method.